packages/relay/src/lib/eth.ts

```diff
@@ -256,6 +257,57 @@ export class EthImpl implements Eth {
     return EthImpl.zeroHex;
   }
 
+  /**
+   * Always returns NO_MINING_WORK error.
+   */
+  getWork(): JsonRpcError {
+    this.logger.trace('getWork()');
+    return Errors['NO_MINING_WORK'];
+  }
+
+  /**
+   * Unsupported methods always return UNSUPPORTED_METHOD error.
+   */
+  submitHashrate(): JsonRpcError {
+    this.logger.trace('submitHashrate()');
+    return Errors['UNSUPPORTED_METHOD'];
```

Comment:
Use predefined here and in all the others
```suggestion
    return predefined.UNSUPPORTED_METHOD;
```